client/src/app/menu/menu.component.html

```diff
@@ -11,31 +11,62 @@
           <div class="logged-in-username">{{ user.username }}</div>
         </div>
 
-        <div class="logged-in-more" ngbDropdown placement="bottom-right auto" container="body">
+        <div class="logged-in-more" ngbDropdown placement="right-top auto" container="body" [autoClose]="'outside'">
           <my-global-icon iconName="more-vertical" ngbDropdownToggle role="button"></my-global-icon>
 
           <div ngbDropdownMenu>
-            <a *ngIf="user.account" [routerLink]="[ '/accounts', user.account.nameWithHost ]" class="dropdown-item">
+            <a *ngIf="user.account" ngbDropdownItem ngbDropdownToggle class="dropdown-item" [routerLink]="[ '/accounts', user.account.nameWithHost ]">
```

Comment:
Why is it a toggle? To close the dropdown when you click on the element?

Reply:
Yes

Reply:
I was forced to make every last-click-element a toggle when I realised the switch had not close the dropdown.

Reply:
Okay then :+1: